IPython/utils/process.py

```diff
@@ -41,6 +41,9 @@ def find_cmd(cmd):
         from IPython.utils.process import pycmd2argv
         argv = pycmd2argv(get_ipython_module_path('IPython.terminal.ipapp'))
 
+    Note, The code for pycmd2argv has been removed now as it was not used
+    anywhere. get_ipython_module_path should give us that same result.
```

Comment:
Thanks @takluyver . Could you please help in one more doubt? I am a newbie to this project and thus have these doubts. Please pardon me for this.

So what I need to do is mention the array as 

   import sys
   argv = [sys.executable, '-m', get_ipython_module_path('IPython.terminal.ipapp')]

Reason I think we need get_ipython_modul_path is that the given module might not be in the same hierarchy mentioned.

Could you please correct me here? Again, thanks a ton for your patience.